contracts/SlashIndicator.template

```diff
@@ -122,7 +122,7 @@ contract SlashIndicator is ISlashIndicator,System,IParamSubscriber{
     4. Note2 that nowhere is a check made to verify that the method caller is himself not the 
        'bad' validator
 */
-  function doubleSignSlash(bytes calldata header1, bytes calldata header2) external onlyInit {
+  function doubleSignSlash(bytes calldata header1, bytes calldata header2) external onlyInit { //@openissue
```

Comment:
this method is currently open for all, allowing the caller to claim rewards. is that what we want?